webapp/channels/src/actions/views/channel.ts

```diff
@@ -535,16 +534,6 @@ export function deleteChannel(channelId: string): ActionFuncAsync<boolean> {
             return {data: false};
         }
 
-        const canViewArchivedChannels = getConfig(state).ExperimentalViewArchivedChannels === 'true';
-        const currentTeamDetails = getCurrentTeam(state);
-        const penultimateViewedChannelName = getPenultimateViewedChannelName(state) ||
```

Comment:
As a followup to this PR, does this let us remove the remaining logic related to the "penultimate viewed channel"? That was added when we were trying to handle cases related to deleting multiple channels in a row so the app wouldn't know where to send you, and since we won't redirect the user after they delete a channel any more, that doesn't seem necessary any more.

As best I can tell by just searching for "penultimate", the only references left to it are to maintain that reference and nothing out side of it seems to be actually using its value any more